sqlc.yaml

```diff
@@ -24,4 +24,12 @@ packages:
     emit_prepared_queries: false
     emit_interface: true
     emit_exact_table_names: false
-    emit_empty_slices: true
\ No newline at end of file
+    emit_empty_slices: true
+    overrides:
+      - db_type: "uuid"
+        go_type:
+          import: "github.com/google/uuid"
+          type: "UUID"
```

Comment:
**question:** why is this override necessary now while it wasn't before?

Reply:
(longer reply: This was a copy-paste from sqlc docs which I didn't remove by accident)